ui/src/app/pages/data/concept/concept-homepage.tsx

```diff
@@ -348,49 +342,51 @@ export const ConceptHomepage = withCurrentWorkspace()(
         if (activeTabSearch) {
-          const conceptDomainCounts = environment.enableNewConceptTabs ? [...resp.domainCounts]
-            : resp.domainCounts.filter(item => item.domain !== Domain.PHYSICALMEASUREMENT);
+          const conceptDomainCounts = environment.enableNewConceptTabs ? resp.domainCounts
+            : resp.domainCounts.filter(item => ![Domain.PHYSICALMEASUREMENT, Domain.SURVEY].includes(item.domain));
           this.setState({
             searchLoading: false,
             conceptDomainCounts: conceptDomainCounts,
-            selectedDomain: resp.domainCounts
-              .find(domainCount => domainCount.domain === cacheItem.domain)});
+            selectedDomain: resp.domainCounts.find(domainCount => domainCount.domain === cacheItem.domain)
+          });
           this.setConceptsAndVocabularies();
         }
       });
       this.setState({selectedConceptDomainMap: selectedConceptDomainMap});
     }
 
-    selectConcepts(concepts: Concept[]) {
-      const {selectedDomain, selectedConceptDomainMap} = this.state;
-      selectedConceptDomainMap[selectedDomain.domain] = concepts.filter(concept => {
-        return concept.domainId.toLowerCase() === selectedDomain.domain.toString().toLowerCase();
-      });
+    selectConcepts(concepts: any[]) {
+      const {selectedDomain: {domain}, selectedConceptDomainMap} = this.state;
+      if (domain === Domain.PHYSICALMEASUREMENT) {
+        selectedConceptDomainMap[domain] = concepts.filter(concept =>
+          concept.domainId.toLowerCase() === Domain[Domain.MEASUREMENT].toLowerCase()
+            && concept.vocabularyId === CriteriaType[CriteriaType.PPI]
+        );
+      } else if (domain === Domain.SURVEY) {
+        selectedConceptDomainMap[domain] = concepts.filter(concept => !!concept.question);
+      } else {
+        selectedConceptDomainMap[domain] = concepts.filter(concept => concept.domainId.toLowerCase() === Domain[domain].toLowerCase());
+      }
       this.setState({selectedConceptDomainMap: selectedConceptDomainMap});
     }
 
     clearSearch() {
       this.setState({
+        currentInputString: '',
         currentSearchString: '',
         showSearchError: false,
         searching: false // reset the search result table to show browse/domain cards instead
       });
     }
 
-    browseDomain(domain: DomainInfo) {
+    browseDomain(domain?: DomainInfo) {
       const {conceptDomainCounts} = this.state;
-      this.setState({browsingSurvey: false, currentSearchString: '',
-        selectedDomain: conceptDomainCounts
-          .find(domainCount => domainCount.domain === domain.domain)},
-        this.searchConcepts);
-    }
-
-    browseSurvey(surveyName) {
-      this.setState({browsingSurvey: true,
-        selectedSurvey: surveyName});
+      const selectedDomain = !domain ? {domain: Domain.SURVEY, name: 'Surveys', conceptCount: 0}
+        : conceptDomainCounts.find(domainCount => domainCount.domain === domain.domain);
```

Comment:
Is it easier to separate domain counts api calls from data api calls from a UI perspective? Seems like it would simplify logic/responsibility of the api call.

Reply:
Yes, that would actually make it simpler on the UI also.